src/components/PageSection.tsx

```diff
@@ -1,4 +1,4 @@
-import { ImageContainer, MainContainer, TextContainer, Title, Body } from './styles/PageSection.styled'
+import { ImageContainer, MainContainer, BufferContainer, TextContainer, Title, Body } from './styles/PageSection.styled'
 
 type Props = {
```

Comment:
can consider making image param optional with default value `''` so that it shows the noimg by default

Reply:
fixed in 8affe34